src/android/IndoorAtlasWayfinding/IAWayfinding.gradle

```diff
@@ -0,0 +1,15 @@
+repositories {
+    mavenCentral()
+    maven {
+        url "https://indooratlas-ltd.bintray.com/mvn-internal"
+        credentials {
```

Comment:
These should not be merged to this public repo: let's wait until the first release in the public Bintray and then change these